jobs/import/templates/bin/run

```diff
@@ -9,7 +9,7 @@ $bin/shield api -k https://<%= p("domain") %> <%= p("core") %>
 echo "login to shield"
 $bin/shield login -u <%= p("username") %> -p <%= p("password") %>
 echo "creating token"
-token=$($bin/shield create-auth-token tmp)
+token=$($bin/shield create-auth-token tmp-$HOSTNAME)
```

Comment:
Rather than use an environment variable, let's use something from `spec`, like a combination of `spec.deployment`, `spec.name`, and `spec.index`.  We could even through in a `$(date +%s)` for good measure, in case previous token cleanup from this errand run fails.